README.md

```diff
@@ -2,12 +2,13 @@
 
 # Summary of what a README file is:
 
-A README file contains important information about the associated source code, files, dependencies and etc., that make up an application. In order to leverage applications and their associated source code, members of the open source community rely on clear and concise READMEs.
+A README file contains important information about the associated source code, files, dependencies and etc. that make up an application. In order to leverage applications and their associated source code, members of the open source community rely on clear and concise READMEs.
```

Comment:
I don't like the use of "and etc." here. Maybe say "associated source code, files, dependencies and related details that make up an application"